packages/new-polymath-ui/src/components/ModalConfirmTransactionQueue/TransactionItem/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { types } from '@polymathnetwork/new-shared';
+import { Box } from '~/components/Box';
+import { IconCircled } from '~/components/IconCircled';
+import { Heading } from '~/components/Heading';
+import { CardPrimary } from '~/components/CardPrimary';
+import { Paragraph } from '~/components/Paragraph';
+import { getTransactionText } from '~/components/utils/contentMappings';
+
+import * as sc from './styles';
+
+interface TransactionItemProps {
+  transaction: types.TransactionEntity;
```

Comment:
```suggestion
  transaction: types.TransactionEntityPojo;
```

Right now they are the same, but it might change in the future, and the Pojo suffixed types are the ones that represent the actual data the components would use, while the ones without the suffix are the normalized entities that we keep in the store